### DISCUSSION TITLE
Hackday planning for September

### DISCUSSION THREAD

**Discussion by bmann** [DWebYVR/notes#35] [Planning & Suggestions]
Starting a new thread for Hackday planning for September.

- a Saturday or Sunday
- figure out The Permanent availability
- who wants to lead this day?
- cowork, hack, teach, demo -- maybe some end of day lightning talks

**Comment by emilymcg** [comment]
**Decision!** Ratified @ cowork day Aug 18 - hackday planning postponed for now. 

Ingredients missing:
- Core organizers (est. 3 required) who are committed to on-the-ground rabble rousing and day-of coordination (Boris and Emily not available through, say, October) 
- Sponsor to cover cost of a weekend day at The Permanent - OR - arrangement for a complimentary weekend day
